src/definitions/modules/dropdown.less

```diff
@@ -1097,7 +1096,7 @@ select.ui.dropdown {
 
   .ui.dropdown > .left.menu .menu,
   .ui.dropdown .menu .left.menu {
-    left: auto;
+    left: auto !important;
```

Comment:
Can we remove `!important` and use repeating selector (e.g. `.left.left.left.menu`) to give more specificity ?
`!important` makes it harder to customise theme.

Reply:
I thought the same first, but i suspect the increased specificity for the other settings (especially `right: 100%`) within that selector will probably cause trouble in other situations then. margin+border are already !important too so to be clean we should then try to remove them as well..

Reply:
Did some quick check and unfortunately the proposed `.left.left.left` does not work and would need adjustments elsewhere too.
Lets face this in a separate PR if needed